www/js/app.js

```diff
@@ -166,7 +165,22 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
     
     // Unique identifier of the article expected to be displayed
     var expectedArticleURLToBeDisplayed = "";
-    
+ 
+    // define and store dark preference for matchMedia
+    var darkPreference = window.matchMedia('(prefers-color-scheme:dark)');
+    // if 'prefers-color-scheme' is not supported in the browser
+    if (window.matchMedia('(prefers-color-scheme)').media === 'not all') {
+        document.getElementById('appThemeSelect').options[0].style.display = "none";
+        document.getElementById('appThemeSelect').options[1].style.display = "none";
```

Comment:
@Maneet-k Did you test this code in IE11 or IE mode? While it matches, setting the display of options 0 and 1 to "none" does nothing at all. They can still be selected in IE11, albeit with the arrow keys (the dropdown doesn't work). If the idea is to remove these options from the UI, then you need to remove them explicitly rather than just hide them. It's also not safe to use `options[0]` (i.e. by number), because in the future we could add more options, and Auto options might not always be the first two. It would be better to match them with `querySelectorAll` perhaps, and then remove them from the DOM, unless you can find a way to hide them that works.